ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/PivotBlockRetriever.java

```diff
@@ -147,9 +126,10 @@ private void handleContestedPivotBlock(final long contestedBlockNumber) {
         contestedBlockNumber, contestedBlockNumber - pivotBlockNumberResetDelta)) {
       LOG.info("Received conflicting pivot blocks for {}.", contestedBlockNumber);
 
-      final int retryCount = confirmationTasks.size();
-      if (retryCount > maxPivotBlockResets
-          || pivotBlockNumber.get() <= BlockHeader.GENESIS_BLOCK_NUMBER) {
+      if (confirmationTasks.size() > SUSPICIOUS_NUMBER_OF_PIVOT_BLOCK_RESETS) {
+        LOG.warn("Several attempts have been made without finding a pivot block");
+      }
+      if (pivotBlockNumber.get() <= BlockHeader.GENESIS_BLOCK_NUMBER) {
         LOG.info("Max retries reached, cancel pivot block download.");
```

Comment:
This is the wrong log message now, it should complain about the selected pivot block being negative or at genesis.  Retries won't trigger this.